Packs/Campaign/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### FindEmailCampaign
+- Added *EmailCampaign.incidents.recipients* and *EmailCampaign.incidents.recipients* outputs which specify the recipients and the domains of the recipients of the incidents found in the campaign.
```

Comment:
-  Added *EmailCampaign.incidents.recipients* output, which specifies the email addresses of the recipients of the incident found in the campaign.
-  Added the *EmailCampaign.incidents.recipients* output, which specifies the domains of the recipients of the incidents found in the campaign.